core/templates/services/exploration-features.service.ts

```diff
@@ -45,12 +45,22 @@ export interface ParamChanges {
   providedIn: 'root'
 })
 export class ExplorationFeaturesService {
+  /**
+   * Static type variable to initialize a service
+   * by setting it to true or false.
+   */
   static serviceIsInitialized = false;
   static settings = {
     areParametersEnabled: false,
     isPlaythroughRecordingEnabled: false
   };
 
+  /**
+   * For enabling the parameters by checking whether the service
+   * has been initialized or not.
```

Comment:
Initialises the service by enabling the parameters feature based on whether the exploration contains parameters.

Reply:
Updated